olive/passes/onnx/conversion.py

```diff
@@ -180,57 +180,73 @@ def _export_pytorch_model(
         if isinstance(pytorch_model, torch.jit.RecursiveScriptModule):
             pytorch_model = TraceModelWrapper(pytorch_model)
 
+        # get input and output names, and dynamic axes
```

Comment:
Is there a reason to move this outside the `if` block? It doesn't look like dynamo exporter requires any of the io_config, other than the dummy imput cleanup. Which in this case could be fixed using the user script if required.

Reply:
I think that the shared implementation of dummy input cleanup keeps the user_script be maintained regardless of export approach. Since output_names and dynamic_axes are used in script export only, let me move them inside the if block.